### PR TITLE
[FIX] web_editor: fetch undraw images in website and note

### PR DESCRIPTION
**Pull request by Goaman**
task-2794153

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/static/tests/tours/media_dialog.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+{
+    trigger: '.s_text_image img',
+    run: "dblclick",
+},
+{
+    trigger: '.form-control.o_we_search',
+    run: "text a",
+},
+{
+    trigger: '.o_we_media_dialog_img_wrapper a[href="https://undraw.co/"]',
```
[the remark this answers is not in the document]
Comment (Goaman) [reply addons/website/static/tests/tours/media_dialog.js:27]: I've change the check method to avoid performing an rpc call.

**Comment by rdeodoo** [comment]
Functionally the Undraw image search is back :+1:

File: addons/website/static/tests/tours/media_dialog.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+wTourUtils.dragNDrop({
+    id: 's_text_image',
+    name: 'Text - Image',
+}),
+{
+    trigger: '.s_text_image img',
+    run: "dblclick",
+},
+{
+    visibilityCheck: false,
```
[the remark this answers is not in the document]
Comment (Goaman) [reply addons/website/static/tests/tours/media_dialog.js:22]: I haven't thought about that. Way better indeed.

File: addons/account/controllers/terms.py [hunk 3]
```diff
@@ -5,9 +5,17 @@
 from odoo.http import request
 
 
+def sitemap_terms(env, rule, qs):
+    if qs and qs.lower() not in '/terms':
```
Comment (xmo-odoo) [inline addons/account/controllers/terms.py:9]: This comparison seems very very odd here, does it really make sense for a qs of `/t` to match?

Also `qs` reads like it's an HTTP query string (which admittedly is a fault of the caller function as well).

Comment (rdeodoo) [reply addons/account/controllers/terms.py:9]: That shouldn't be in this PR, there was probably an error when he rebased the branch, it seems to have been including some other commits :grimacing: 

But yes, when the user search `/t` or `/te` etc when creating a link, this page should be returned.

image

Comment (Goaman) [reply addons/account/controllers/terms.py]: Indeed, there was a problem with a rebase, that commit does not belong to this PR 😬.

**Comment by rdeodoo** [comment]
@Goaman Any idea when this will be shipped?

Wanted to showcase the feature today and still not working :'(

**Comment by dmo-odoo** [comment]
@rdeodoo This was currently blocked in Testing IN phase because the tester encountered a traceback, but I just checked and the same traceback can be triggered in the exact same way on runbot 15.0 and is therefore irrelevant to this task.

@robodoo r+

**Comment by rdeodoo** [comment]
It is never too late or too soon. It is when it is supposed to be.

**Comment by dmo-odoo** [comment]
🧙‍♂️

**Pull request closed without merging**